ci/test/00_setup_env_mac_host.sh

```diff
@@ -10,7 +10,8 @@ export HOST=x86_64-apple-darwin18
 export PIP_PACKAGES="zmq"
 export GOAL="install"
 # ELEMENTS: add -fno-stack-check to work around clang bug on macos
-export BITCOIN_CONFIG="--with-gui --enable-reduce-exports --enable-werror --with-boost-process CXXFLAGS=-fno-stack-check"
+# ELEMENTS: remove --enable-werror because it triggers on Boost Thread includes (FIXME remove this after 22.0 rebase when boost-thread is removed)
```

Comment:
Remembering this will be a non-trivial task :sweat_smile:

Reply:
In the 22 rebase I just blanked out this PR, since the PRs it backports preceded it.